grommet-leaflet/src/layers/Map/Map.jsx

```diff
@@ -57,9 +65,11 @@ Map.propTypes = {
   center: PropTypes.array,
   children: PropTypes.any,
   scrollWheelZoom: PropTypes.string || PropTypes.bool,
+  maxZoom: PropTypes.number,
   tileLayer: PropTypes.shape({
     attribution: PropTypes.string,
     url: PropTypes.string.isRequired,
+    format: PropTypes.oneOf(['pmtiles']),
```

Comment:
Attribution is marked as required in typescript definition, but not here. Just need to align them.